src/pybind/mgr/cephadm/templates/services/ingress/keepalived.conf.j2

```diff
@@ -11,19 +11,21 @@ vrrp_script check_backend {
 vrrp_instance VI_{{ x }} {
   state {{ states[x] }}
   priority {{ priorities[x] }}
-  interface {{ interface }}
-  virtual_router_id {{ 50 + x }}
+  interface {{ vrrp_interface }}
+  virtual_router_id {{ first_virtual_router_id + x }}
   advert_int 1
   authentication {
       auth_type PASS
       auth_pass {{ password }}
   }
+{% if not spec.use_keepalived_multicast %}
```

Comment:
So, if this `use_keepalived_multicast` is set and we leave out this block, is it assumed keepalived will see this missing and automatically make use of the multicast IP? I see we never explicitly set anything in this case, just leave things out.

Reply:
It is correct. We do not need to specify anything for keepalived to work with multicast. It will just use the interface set on the config file.